src/PWAdditive.cpp

```diff
@@ -55,8 +55,8 @@ DDWAF_RET_CODE PWAdditive::run(ddwaf_object newParameters,
     // consistent across all possible timeout scenarios.
     if (timeLeft == 0)
     {
-        if (res) {
-            ddwaf_result &output = res.value();
+        if (res.has_value()) {
```

Comment:
If you set the minimum version but don't actually make these changes does it still build in CI? Because if that is the case we don't really need them.

Reply:
It doesn't and fails with the following message: 
```
    /private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/pip-req-build-885l7kqt/build/temp.macosx-10.9-x86_64-3.6/ddtrace.appsec._ddwaf_0/libddwaf-prefix/src/libddwaf/src/PWAdditive.cpp:84:36: error: 'value' is unavailable: introduced in macOS 10.14
            ddwaf_result &output = res.value();
```

Link to the full build logs:
https://github.com/DataDog/dd-trace-py/runs/4302682573?check_suite_focus=true#step:5:878